nums/core/array/base.py

```diff
@@ -172,7 +172,13 @@ def _block_from_other(self, other):
             False,
             self._cm,
         )
-        block.oid = self._cm.put(np.array(other, dtype=self.dtype))
+        block.oid = self._cm.put(
+            np.array(other, dtype=self.dtype),
+            syskwargs={
```

Comment:
Let's add a comment here justifying the `syskwargs` of `other` here.

Reply:
We want to justify the choice of `syskwargs`, namely, `other` will be a literal that we want local to self. Can you update accordingly? Please use punctuation with all comments.